superset/commands/importers/v1/__init__.py

```diff
@@ -67,6 +67,9 @@ def run(self) -> None:
         try:
             self._import(db.session, self._configs, self.overwrite)
             db.session.commit()
+        except CommandException as ex:
+            db.session.rollback()
+            raise ex
         except Exception as ex:
             db.session.rollback()
             raise self.import_error() from ex
```

Comment:
`from ex` will raise the generic dataset import error with the `ex` message on the `__cause__` attr. This cause will not be included on the API error message response.